packages/orchestrator-ui-components/src/pages/startPage/WfoStartPage.tsx

```diff
@@ -111,7 +120,27 @@ export const WfoStartPage = () => {
             name: t('activeWorkflows.buttonText'),
             url: PATH_WORKFLOWS,
         },
-        isLoading: processesSummaryIsFetching,
+        isLoading: activeWorkflowsSummaryIsFetching,
+    };
+
+    const myWorkflowsSummaryCard: SummaryCard = {
+        headerTitle: t('myWorkflows.headerTitle'),
+        headerValue: myWorkflowsSummaryResponse?.pageInfo.totalItems ?? 0,
+        headerStatus: SummaryCardStatus.Success,
+        listTitle: t('myWorkflows.listTitle'),
+        listItems:
+            myWorkflowsSummaryResponse?.processes.map((workflow) => ({
+                title: workflow.workflowName,
+                value: formatDate(workflow?.startedAt),
+                url: `${PATH_WORKFLOWS}/${workflow.processId}`,
+            })) ?? [],
+        button: username
+            ? {
```

Comment:
I am in doubt whether we should hide the summary card if oath is disabled. I guess most deployments will have oath enabled, but for the example-orchestrator-ui, this card will useless...

Reply:
Done